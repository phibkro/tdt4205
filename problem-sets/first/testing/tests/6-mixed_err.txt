10:go:dx
dx= // Comment
go go goooo
dy=11dx=11
-5: go
go dx=11//Comment
go / Comment
8: dy=42 dx=4.2
I beheld the wretchâ€”the miserable monster whom I had created
apple pie
