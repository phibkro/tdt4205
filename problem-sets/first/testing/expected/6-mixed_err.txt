Table filled!
line    1: error: 10:go:dx
~~~~~~~~~~~~~~~~~~~~~^
line    2: error: dx= // Comment
~~~~~~~~~~~~~~~~~~~~~^
line    3: error: go go goooo
~~~~~~~~~~~~~~~~~~~~~~~~~~^
line    4: error: dy=11dx=11
~~~~~~~~~~~~~~~~~~~~~~~^
line    5: error: -5: go
~~~~~~~~~~~~~~~~~~^
line    6: error: go dx=11//Comment
~~~~~~~~~~~~~~~~~~~~~~~~~~^
line    7: error: go / Comment
~~~~~~~~~~~~~~~~~~~~~~^
line    8: error: 8: dy=42 dx=4.2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
line    9: error: I beheld the wretchâ€”the miserable monster whom I had created
~~~~~~~~~~~~~~~~~~^
line   10: error: apple pie
~~~~~~~~~~~~~~~~~~^
